.github/workflows/themes-weekly-pull.yml

```diff
@@ -0,0 +1,46 @@
+name: themes-weekly-pull
+
+on:
+  schedule:
+    # run every friday at 5AM
+    - cron: '0 5 * * 5'
+
+jobs:
+  pull-from-jsonbin:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install Node.js 14.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install dependencies
+        run: yarn
+
+      - name: Run themes:pull script
+        run: yarn run themes:pull
+        env:
+          MASTER_KEY: ${{ secrets.MASTER_KEY }}
+          BIN_ID: ${{ secrets.BIN_ID }}
+
+      - name: Generate dark tokens
+        run: npx token-transformer scripts/themes/tokens/arduino-tokens.json scripts/themes/tokens/dark.json core,ide-default,ide-dark,theia core,ide-default,ide-dark
+
+      - name: Generate default tokens
+        run: npx token-transformer scripts/themes/tokens/arduino-tokens.json scripts/themes/tokens/default.json core,ide-default,theia core,ide-default
+
+      - name: Run themes:generate script
+        run: yarn run themes:generate
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          commit-message: Updated themes
+          title: Update themes
+          branch: themes/themes-update
+          author: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>
+          base: ${{ github.head_ref }}

```

Comment:
```suggestion
```
`github.head_ref` is an empty string when the workflow run is triggered by a `schedule` event.

It should be fine to remove it because:

https://github.com/peter-evans/create-pull-request#action-inputs

>base	Sets the pull request base branch.	Defaults to the branch checked out in the workflow.

https://github.com/actions/checkout#usage

> ```yaml
>     # The branch, tag or SHA to checkout. When checking out the repository that
>     # triggered a workflow, this defaults to the reference or SHA for that event.
>     # Otherwise, uses the default branch.
>     ref: ''
> ```

The `schedule` event only runs from the default branch, so this means `peter-evans/create-pull-request` will use `main` as the base of the PR.

Reply:
Very clear, thanks for the suggestion.